google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractStructReader.java

```diff
@@ -61,6 +61,8 @@ protected String getPgJsonbInternal(int columnIndex) {
     throw new UnsupportedOperationException("Not implemented");
   }
 
+  protected abstract long getPgOidInternal(int columnIndex);
```

Comment:
Add a default implementation for this and the other methods that are being added to this class. The default implementation should just throw UnsupportedOperationException. This prevents this change from being a breaking change, as anyone who has a class that extends AbstractStructReader does not get a compile error.

That should also remove the requirement to add the method names to clirr-ignored-differences.xml

Reply:
Done.